.travis/build_computecpp.sh

```diff
@@ -5,7 +5,8 @@ set -ev
 ###########################
 # Get ComputeCpp
 ###########################
-wget --no-check-certificate https://computecpp.codeplay.com/downloads/computecpp-ce/0.5.0/ubuntu-14.04-64bit.tar.gz
+wget --no-check-certificate https://computecpp.codeplay.com/downloads/computecpp-ce/0.5.1/ubuntu-14.04-64bit.tar.gz
 tar -xzf ubuntu-14.04-64bit.tar.gz -C /tmp
+mv /tmp/ComputeCpp-CE-0.5.1-Ubuntu-14.04-64bit /tmp/computecpp
```

Comment:
why don't you import the new Khronos SYCL Parallel STL travis script? It also test with Intel OpenCL when available

Reply:
OK, I will add this in a separate CL I think. Thanks for the suggestion.